syntax = "proto3";

package hello;

service HelloService {
    rpc SayHello (HelloRequest) returns (HelloResponse);
}

service FraudService {
    rpc FraudName (NameRequest) returns (NameResponse);
    rpc FraudExp (ExpRequest) returns (ExpResponse);
}

message NameRequest{
    string name = 1;
}

message ExpRequest {
    string ExpDate = 1;
}

message NameResponse{
    bool status2 = 1;
}

message ExpResponse {
    bool status = 1;
}

message HelloRequest {
    string name = 1;
}

message HelloResponse {
    string greeting = 1;
}
